src/parser/shared/modules/items/Healthstone.js

```diff
@@ -1,18 +1,37 @@
 import SPELLS from 'common/SPELLS/index';
 
+import Combatants from 'parser/shared/modules/Combatants';
+import Abilities from 'parser/core/modules/Abilities';
+import Buffs from 'parser/core/modules/Buffs';
+import SpellUsable from 'parser/shared/modules/SpellUsable';
+import AbilityTracker from 'parser/shared/modules/AbilityTracker';
+import { Class } from 'parser/core/Events';
 import Potion from './Potion';
-
 /**
  * Tracks Healthstone cooldown.
  */
 class Healthstone extends Potion {
+  // Potion does not use Combatants, other dependencies are inherit from Potion.
+  static dependencies = {
+    abilities: Abilities,
+    buffs: Buffs,
+    spellUsable: SpellUsable,
+    abilityTracker: AbilityTracker,
+    combatants: Combatants,
+  };
+
   static spells = [
     SPELLS.HEALTHSTONE,
   ];
   static recommendedEfficiency = 0;
   static extraAbilityInfo = {
     isDefensive: true,
   };
+
+  get isAvailable() {
+    const players = Object.values(this.combatants.players);
+    return players.find(combatant => (combatant.spec.className === Class.Warlock));
```

Comment:
Is `some` more intuitive than `find` here? Non-blocking of the PR, I personally just prefer to see something that doesn't rely on truthyness

Reply:
What is the main difference between `find` and `some`?

Reply:
`find` returns the found value, `some` returns a boolean -- your arrow function wouldn't even have to change I think

Reply:
That seems to be better. Since it now returns either `undefined` or `Combatant`. Would also be typescript proof.